app/pages/home-logged-in.cjsx

```diff
@@ -84,7 +84,7 @@ RecentProjects = React.createClass
             title: "home.recentProjects.title"
             href: "/users/#{@props.user.login}/stats"
             action: "home.recentProjects.button"
-          projectsPromise = apiClient.type('projects').get userProjects, include: ['avatar', 'owners']
+          projectsPromise = apiClient.type('projects').get(id: userProjects, cards: true, include: ['avatar', 'owners'])
         else
```

Comment:
I think the cards endpoint includes the avatars and owners, so you can get rid of the `include` parameter here.